src/thermal.cpp

```diff
@@ -89,9 +90,12 @@ ThermalScattering::ThermalScattering(
           temps_to_read.push_back(std::round(temp_actual));
         }
       } else {
-        fatal_error(fmt::format("Nuclear data library does not contain cross "
-                                "sections for {} at or near {} K.",
-          name_, std::round(T)));
+        fatal_error(fmt::format(
+          "Nuclear data library does not contain cross sections "
+          "for {}  at or near {} K. Available temperatures "
+          "are {} K consider making use of openmc.Settings.temperature "
+          "to specify how intermediate temperatures are treated.",
+          name_, std::to_string(std::round(T)), concatenate(temps_available)));
```

Comment:
`std::to_string` shouldn't be necessary here I don't think.

```suggestion
          name_, std::round(T), concatenate(temps_available)));
```

Same in the message above.

Reply:
Solved.